website/snippets/_git-providers-supporting-ci.md

```diff
@@ -0,0 +1,8 @@
+If your git provider has a native dbt Cloud integration, you can set up [continuous integration (CI) job](/docs/deploy/ci-jobs) within dbt Cloud. For providers without native integration, you can still use a Git URL. The following table explains the available integration options and their capabilities.
```

Comment:
also for this snippet, to address Joel's issue and [my suggestion](https://www.notion.so/dbtlabs/Clarify-what-git-providers-support-CI-160bb38ebda780c090ddec8a3bbbfb1d?pvs=4#161bb38ebda7802cb985fade7084b021) -- should this snippet table also be used in the git configuration page (e.g. maybe the landing page or git url page)?

>t'd be useful to just have that listed in the CI docs as well as the configure git docs